acronym/acronym_test.rb

```diff
@@ -0,0 +1,19 @@
+
+class Arguments
+  def self.run(arg_v)
+    for i in 0 ... arg_v.length
+        puts "#{i} #{arg_v[i]}"
+     end
+  end
+end
+
+class Acronym
+    def abbreviate
+        ARGV[0].split("-").join(" ").split.map(&:chr).join.upcase
```

Comment:
Let's also try to handle the following cases: 
String can contain symbols like &, %, $, — etc. example:
 1. input: 'Ernst&Yong'
     output: 'EY'
 2. input: 'Something — I made up from thin air'
     output: 'SIMUFTA'
 3. input: 'First In,First Out'
      output: 'FIFO'
So any symbol except a-z A-Z and spaces should be treated as separator and be omitted from acronym

Reply:
These characters (like &, %, $) are ignored and only the first letters of each word are displayed, like FIFO. You can check it yourself.